 Header header
 string id
 handBlobTracker/HFPose2D[] measurements
 bool[] valid
